### PR TITLE
Update API Fields

### PR DESCRIPTION
**Pull request by Miner28**
Added Recommended Capacity to World
Removed fallbackAvatar from LimitedUser

### CODE REVIEW

File: openapi/components/schemas/World.yaml [hunk 1]
```diff
@@ -9,9 +9,13 @@ properties:
     type: string
   capacity:
     example: 8
-    maximum: 40
```
Comment (Rexios80) [inline openapi/components/schemas/World.yaml:12]: Not sure we should be specifying a min/max for a value we don't control. What is the benefit of this?

File: openapi/components/schemas/World.yaml [hunk 2]
```diff
@@ -9,9 +9,13 @@ properties:
...
+    maximum: 80
     minimum: 0
     type: integer
+  recommendedCapacity:
+    example: 4
+    maximum: 80
```
Comment (Rexios80) [inline openapi/components/schemas/World.yaml:17]: Same here

File: openapi/components/schemas/LimitedUser.yaml [hunk 3]
```diff
@@ -48,7 +48,6 @@ required:
   - statusDescription
   - currentAvatarImageUrl
   - currentAvatarThumbnailImageUrl
-  - fallbackAvatar
```
Comment (ariesclark) [inline openapi/components/schemas/LimitedUser.yaml:51]: This is still present within the API, why is it being removed?

File: openapi/components/schemas/World.yaml (hunk 1 above)
Comment (ariesclark) [reply openapi/components/schemas/World.yaml:12]: I agree with not including arbitrary limits, some generators may enforce these while others don't.

**Review by ariesclark — CHANGES_REQUESTED** [review]

File: openapi/components/schemas/LimitedUser.yaml (hunk 3 above)
Comment (Rexios80) [reply openapi/components/schemas/LimitedUser.yaml:51]: Maybe it doesn't come back on all requests anymore? They just removed it from the required fields.

Comment (ariesclark) [reply openapi/components/schemas/LimitedUser.yaml:51]: If that's the case, please provide under which conditions it is not included.

Comment (Miner28) [reply openapi/components/schemas/LimitedUser.yaml:51]: Its not @ariesclark

Comment (Miner28) [reply openapi/components/schemas/LimitedUser.yaml:51]: > If that's the case, please provide under which conditions it is not included. I cannot reproduce it's removal.

Idk what you're doing its not there under any endpoint for me

**Review by ariesclark — APPROVED** [review]

**Pull request merged**